src/plugins/intel_gpu/include/intel_gpu/primitives/prior_box8.hpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include <cmath>
+#include <limits>
+#include <vector>
+
+#include "primitive.hpp"
+
+namespace cldnn {
+/// @addtogroup cpp_api C++ API
+/// @{
+/// @addtogroup cpp_topology Network Topology
+/// @{
+/// @addtogroup cpp_primitives Primitives
+/// @{
+
+/// @brief Generates a set of default bounding boxes with different sizes and aspect ratios.
+/// @details The prior-boxes are shared across all the images in a batch (since they have the same width and height).
+/// First feature stores the mean of each prior coordinate.
+/// Second feature stores the variance of each prior coordinate.
+struct prior_box8 : public primitive_base<prior_box8> {
+    struct Attributes {
```

Comment:
lowercase to be aligned with naming style of this file

Reply:
Done